topics/admin/tutorials/beacon/tutorial.md

```diff
@@ -60,9 +60,9 @@ This tutorial will guide you through setting up a [GA4GH Beacon](https://beacon-
 >    ```diff
 >    --- a/hosts
 >    +++ b/hosts
->    @@ -6,3 +6,12 @@ galaxyservers
->     gat-0.au.training.galaxyproject.eu ansible_user=ubuntu
->     [monitoring]
+>    @@ -9,3 +9,12 @@ gat-0.eu.training.galaxyproject.eu ansible_connection=local ansible_user=ubuntu
+>     
+>     [sentryservers]
```

Comment:
I guess that diff isn't quite right ?

Reply:
ah this is in beacon? I think this is OK, shows up with sentry now because we're sticking it before beacon.